src/screens/groups/components/batch-recipients-list/batch-recipient-info.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Header, Table, Checkbox, Input } from 'semantic-ui-react'
+import type { RecipientDTO } from '@/lib/dto/RecipientDTO'
+import type { SelectedRecipients } from '@/screens/groups/batch-payment-screen'
+
+interface Props {
+  recipient: RecipientDTO
+  selectedRecipients: SelectedRecipients
+  handleRowSelect: (_id: string) => void
+  handleAmountChange: (
+    _id: string,
+    _event: React.ChangeEvent<HTMLInputElement>
+  ) => void
+}
+
+const BatchRecipientInfo = ({
+  recipient,
+  selectedRecipients,
+  handleRowSelect,
+  handleAmountChange,
+}: Props) => (
+  <>
```

Comment:
We can remove this fragment, because `<Table.Row>` wraps all the following text.